### PR TITLE
A bunch of major changes

### PR DESCRIPTION
**Pull request by ryannealmes**
- Cleaned up a bunch of files 
- Fixed creation and deletion and added specs as well
- Removed configuration for now. It can be added when we have more of the actual functionality working

It's been a while since I have written ruby so I should manage smaller PRs as I go :)

### CODE REVIEW

File: lib/enum_weasel/generators/tables/creation.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+        def initialize(model_enums, database_enum_tables)
+          @model_enums = model_enums
+          @database_enum_tables = database_enum_tables
+          @connection = ActiveRecord::Base.connection
+        end
+
+        def call
+          new_enum_tables.each do |table|
+            creat_table table
```
Comment (atainton) [inline lib/enum_weasel/generators/tables/creation.rb]: spelling

File: lib/enum_weasel/generators/tables/deletion.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          database_enum_tables - formatted_model_enum_names
+        end
+
+        def delete_table enum_table
+          if connection.table_exists?(enum_table)
+            connection.drop_table enum_table 
+          end
+        end
+
+        def formatted_model_enum_names
```
Comment (atainton) [inline lib/enum_weasel/generators/tables/deletion.rb]: this is repeated in both classes may be worthwhile moving this into a parent class

File: .eslint [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (atainton) [inline .eslint]: what is this file used for?

**Review by atainton — COMMENTED** [review]
this is looking really good, I would def like to add the data to the tables will try do that this week. There are 3 comments

**Comment by ryannealmes** [comment]
I wasn't too keen on refactoring duplication yet, just as it may be a bit early still. Anyway, I decided just to do it and it actually looks pretty good. Let me know if there are any other changes you want. Going to checkout rubocop now.

**Comment by ryannealmes** [comment]
Pushed up some rubocop changes. There are still a couple outstanding, but my head started to hurt :) Some of them don't make sense so I think disabling some might be a good idea as well. Let's find the right balance.

**Comment by atainton** [comment]
👍

**Pull request merged**